### ISSUE TITLE
Trend metric does not work when a suffix is used

### ISSUE THREAD

**Issue by nick-jansen** [laravel/nova-issues#3263]
- Laravel Version: v8.29.0
- Nova Version: v3.22.0
- PHP Version: v7.4
- Database Driver & Version: MySQL
- Operating System and Version: Ubuntu 20.04
- Browser type and version: Chrome 88

### Description:

Trend metric does not work when a suffix is used (after upgrading to Nova v3.22.0 and publishing assets).

![Schermafbeelding 2021-02-24 om 08 14 24](https://user-images.githubusercontent.com/20165378/108961971-60f8c000-7678-11eb-9910-55a875315816.png)

![Schermafbeelding 2021-02-24 om 08 17 11](https://user-images.githubusercontent.com/20165378/108962221-bcc34900-7678-11eb-8f7e-785dddb3736d.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create a new Trend metric and add a suffix to the calculated value.

```php
public function calculate(NovaRequest $request)
{
    return $this->sumByDays($request, ProjectHour::where('user_id', Auth::user()->id), 'hours', 'date')
        ->suffix('hours')
        ->showLatestValue();
}
```

**Comment by AshleyMinshall** [comment]
Why was this closed? What is the solution?

**Comment by crynobone** [comment]
> Why was this closed? What is the solution?

Closed **assigned** with labeled as **bug** generally means a PR has been merged. However, we normally only update this issue after a new release.

**Comment by AshleyMinshall** [comment]
I see, so has it been merged? I looked at the releases for nova releases and the only mentions of suffix issues are old. How should I proceed? 

NOTE: I have done `php artisan nova:publish` and I'm on 3.22.0.

Thanks
Ashley

**Comment by crynobone** [comment]
Again, we haven't update this issue since no release is available yet.

**Comment by AshleyMinshall** [comment]
Ok, I don't fully understand what you mean but I am assuming that:
- Its not fixed right now
- It will be in the next release?

If so is there an ETA? 

Am I missing something @nick-jansen ? Did you find a solution? I ended up replacing suffix -> prefix, but it looks weird for client.

Thanks
Ashley

**Comment by crynobone** [comment]
This has been fixed in 3.23.0 but please upgrade to 3.23.1. https://nova.laravel.com/releases
